tests/data/test_optimization_indexset.py

```diff
@@ -34,157 +34,160 @@ def df_from_list(indexsets: list):
     )
 
 
-@all_platforms
 class TestDataOptimizationIndexSet:
-    def test_create_indexset(self, test_mp, request):
-        test_mp: Platform = request.getfixturevalue(test_mp)  # type: ignore
-        run = test_mp.backend.runs.create("Model", "Scenario")
-        indexset_1 = test_mp.backend.optimization.indexsets.create(
+    def test_create_indexset(self, platform: ixmp4.Platform):
+        run = platform.backend.runs.create("Model", "Scenario")
+        indexset_1 = platform.backend.optimization.indexsets.create(
             run_id=run.id, name="Indexset"
         )
         assert indexset_1.id == 1
         assert indexset_1.run__id == 1
         assert indexset_1.name == "Indexset"
 
         with pytest.raises(IndexSet.NotUnique):
-            _ = test_mp.backend.optimization.indexsets.create(
+            _ = platform.backend.optimization.indexsets.create(
                 run_id=run.id, name="Indexset"
             )
 
-    def test_get_indexset(self, test_mp, request):
-        test_mp: Platform = request.getfixturevalue(test_mp)  # type: ignore
-        run = test_mp.backend.runs.create("Model", "Scenario")
-        create_indexsets_for_run(platform=test_mp, run_id=run.id, amount=1)
-        indexset = test_mp.backend.optimization.indexsets.get(
+    def test_get_indexset(self, platform: ixmp4.Platform):
+        run = platform.backend.runs.create("Model", "Scenario")
+        create_indexsets_for_run(platform=platform, run_id=run.id, amount=1)
+        indexset = platform.backend.optimization.indexsets.get(
             run_id=run.id, name="Indexset 1"
         )
         assert indexset.id == 1
         assert indexset.run__id == 1
         assert indexset.name == "Indexset 1"
 
         with pytest.raises(IndexSet.NotFound):
-            _ = test_mp.backend.optimization.indexsets.get(
+            _ = platform.backend.optimization.indexsets.get(
                 run_id=run.id, name="Indexset 2"
             )
 
-    def test_list_indexsets(self, test_mp, request):
-        test_mp: Platform = request.getfixturevalue(test_mp)  # type: ignore
-        run = test_mp.backend.runs.create("Model", "Scenario")
+    def test_list_indexsets(self, platform: ixmp4.Platform):
+        run = platform.backend.runs.create("Model", "Scenario")
+        # Per default, list() lists scalars for `default` version runs:
+        platform.backend.runs.set_as_default_version(run.id)
         indexset_1, indexset_2 = create_indexsets_for_run(
-            platform=test_mp, run_id=run.id
+            platform=platform, run_id=run.id
         )
-        assert [indexset_1] == test_mp.backend.optimization.indexsets.list(
-            name=indexset_1.name
+        assert [indexset_1] == platform.backend.optimization.indexsets.list(
+            name="Indexset 1"
```

Comment:
Why can this not be `indexset_1.name`?
Same below in similar places.

Reply:
I missed that one in the conflict res.